docs/packages/patches.json

```diff
@@ -0,0 +1,9 @@
+{
+  "patches" : [
+    {
+      "file" : "Thrust/cub_odr.diff",
+      "issue" : "cub kernel dispatch ODR [https://github.com/NVIDIA/cub/issues/545]",
```

Comment:
I'd merge this PR but remove the specific patches (CUB ODR). Just make this PR the infrastructure.

Reply:
Ah, sorry. I didn't catch that this was in `docs/`.